docs/user-manual/civiform-admin-guide/manage-questions.md

```diff
@@ -63,22 +78,20 @@ When you restore an archived question, you can use it in your programs. You can
-| Radio Button | <p>Suitable for a short list (&#x3C;=7 items) of static items where the Applicant is required to select only one option. For example, simple yes/no questions or employment status.</p><p><strong>Tip</strong>: If you want Applicants to select multiple options in a question, use a Checkbox question instead.</p>                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
-| Static Text  | <p>A free form field that includes the ability to fully format text using Markdown. See <a href="using-markdown.md">Using Markdown</a> for more information.|
-
-| Text         | A free form field that can store letters, numbers, characters, or symbols.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
-|Phone         | <p>It accepts two inputs from the user: the country, and the number. The number is a formatted input in the (xxx) xxx-xxxx format. Currently, it supports only US and Canadian numbers. The phone numbers are validated and stored as Strings. When the admin sees the entered number, they see it as +1 xxx-xxx-xxxx format.</p>
+| Address | An Applicant’s address. For example, residential, work, mailing, school, etc. |
```

Comment:
question (non-blocking): Did you mean to make these changes? I think it's fine, but it does change some of the formatting.